### PR TITLE
EWPP-812: Add heading to timeline pattern.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: oe_theme.theme [hunk 1]
```diff
@@ -1361,6 +1361,9 @@ function oe_theme_preprocess_paragraph__oe_facts_figures(array &$variables): voi
  */
 function oe_theme_preprocess_paragraph__oe_timeline(array &$variables): void {
   $paragraph = $variables['paragraph'];
+  if (!$paragraph->get('field_oe_title')->isEmpty()) {
```
Comment (upchuk) [inline oe_theme.theme]: Let's put a check here if the field exists in case oe theme is updated while oe_paragraphs doesn't yet get the field on a site.

**Review by upchuk — CHANGES_REQUESTED** [review]

**Review by upchuk — CHANGES_REQUESTED** [review]
Approving but blocking the merge for the full epic.

File: tests/Kernel/fixtures/rendering.yml [hunk 2]
```diff
@@ -2215,6 +2216,7 @@
           body: "Lorem Ipsum 6"
   assertions:
     count:
+      'h2.ecl-u-type-heading-2': 1
```
Comment (sergepavle) [inline tests/Kernel/fixtures/rendering.yml:2222]: I think it would be nice in another test to check that we don't have this h2 in case the title is empty.

**Review by sergepavle — CHANGES_REQUESTED** [review]

**Review by sergepavle — DISMISSED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by upchuk — APPROVED** [review]
